python.md

```diff
@@ -2,8 +2,54 @@
 layout: page
 title: Python
 ---
-TODO
+
+# General Tips
+
+* Try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/), except for line length. Use a linter (like [pylint](http://www.pylint.org/)) to enforce them automatically in your code.
```

Comment:
Kinda prefer the links setup the [frontend.md](https://raw.githubusercontent.com/fieldaware/style-guide/gh-pages/frontend.md) section has.